pbs_scripts/sra_to_salmon_ccul.script

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -l
+
+################################################################################
+#Script Name: sra_to_salmon_pa14.pbs
+#Description: Submission script for sra_to_salmon job arrays
+#Args       : None 
+#Author     : Georgia Doing
+#Email      : Georgia.Doing.GR@Dartmouth.edu
+################################################################################
+
+##PBS -q largeq
+##PBS -A NCCC
+##PBS -N s2sO1
+##PBS -l walltime=15:00:00
+##PBS -l nodes=1:ppn=6
+##PBS -m bea
+##PBS -M Georgia.Doing.GR@Dartmouth.edu
```

Comment:
Similar idea here. Not sure if you'd rather specify this in a readme to the user.